dev-tools/compose/docker-compose.yml

```diff
@@ -41,7 +41,7 @@ services:
   db:
     image: postgres:11.4
     shm_size: 2g
-    command: postgres -c shared_buffers=2GB
+    command: postgres -c shared_buffers=2GB -c max_connections=300
```

Comment:
did you hit this? we should probably go higher since *all* services use this same db. maybe 1000

Reply:
yeah i hit it hard - after changing i wasn't seeing more than 150 though